server/routes/login.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+router.post('/api/login', (req, res) => {
+  const { email, password } = req.body;
+  // Here we would typically check the credentials against a database
```

Comment:
Although it's mentioned that for simplicity, any credentials are accepted, it's recommended to implement some sort of mock authentication (like a hardcoded username and password) to mimic the real-world scenario.